modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro

```diff
@@ -681,6 +683,56 @@ definition {
 				key_fieldName = "activities[${activityIndex}].activityDescription.hiringOutsideWriterOrAgency",
 				locator1 = "PRMMDFRequest#INPUT_BUTTON");
 		}
+		
+		if ((${typeOfActivity} == "Digital Marketing") && (${tactic} == "Email Campaign")) {
+			Type(
+				key_fieldName = "activities[${activityIndex}].activityDescription.overallMessageContentCTA",
+				locator1 = "PRMMDFRequest#TEXT_FIELD",
+				value1 = "Overall Message");
+
+			Type(
+				key_fieldName = "activities[${activityIndex}].activityDescription.landingPageCopy",
+				locator1 = "PRMMDFRequest#TEXT_FIELD",
+				value1 = "Landing Page");
+
+			if (isSet(nurtureDripCampaign)) {
+				Click.clickAtNotVisible(
+					key_buttonText = ${nurtureDripCampaign},
+					key_fieldName = "activities[${activityIndex}].activityDescription.nurtureDripCampaign",
+					locator1 = "PRMMDFRequest#INPUT_BUTTON");
+			}
+
+			if (${nurtureDripCampaign} == "Yes") {
+				Type(
+					key_fieldName = "activities[${activityIndex}].activityDescription.manySeries",
+					locator1 = "PRMMDFRequest#TEXT_FIELD",
+					value1 = "Many Series");
+			}
```

Comment:
Hi @marinaasant0s, this condition should be inside the previous condition.
The logic should be something like:
`If nurtureDripCampaign` is set, then select the input button as `yes` or `no`.
   ` If input button == yes,` then `fill the field`.